OptimizelySDK.Tests/DecisionServiceTest.cs

```diff
@@ -318,5 +318,38 @@ public void GetVariationSavesANewUserProfile()
             Assert.IsTrue(TestData.CompareObjects(variation, decisionService.GetVariation(experiment, UserProfileId, new UserAttributes())));
             UserProfileServiceMock.Verify(_ => _.Save(It.IsAny<Dictionary<string, object>>()), Times.Once);
         }
+
+        [Test]
+        public void TestGetVariationUserWithSetForcedVariation()
+        {
+            var experimentKey = "test_experiment";
+            var pausedExperimentKey = "paused_experiment";
+            var userId = "test_user";
+            var forcedVariationKey = "variation";
+            var bucketedVariationKey = "control";
+            var optlyObject = new Optimizely(TestData.Datafile, new ValidEventDispatcher(), LoggerMock.Object);
+            var userAttributes = new UserAttributes
+            {
+                {"device_type", "iPhone" },
+                {"location", "San Francisco" }
+            };
+            
+            optlyObject.Activate(experimentKey, userId, userAttributes);
+            // confirm normal bucketing occurs before setting the forced variation
+            forcedVariationKey = optlyObject.GetVariation(experimentKey, userId, userAttributes);
+            Assert.AreEqual(bucketedVariationKey, forcedVariationKey);
+            // test valid experiment
+            Assert.IsTrue(optlyObject.SetForcedVariation(experimentKey, userId, forcedVariationKey), string.Format(@"Set variation to ""{0}"" failed.", forcedVariationKey));
+            forcedVariationKey = optlyObject.GetVariation(experimentKey, userId, userAttributes);
```

Comment:
Declare it as var actualForcedVariationKey.

Reply:
Done